src/parser.js

```diff
@@ -251,6 +251,9 @@ export default class Parser extends Stream {
               if (entry.attributes['INSTREAM-ID']) {
                 rendition.instreamId = entry.attributes['INSTREAM-ID'];
               }
+              if (entry.attributes['CHARACTERISTICS']) {
```

Comment:
I was just trying to be consistent with the pattern used for `INSTREAM-ID` already. Are you saying you want me to fix that, as well as what I added?